src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,38 @@
+package duke;
+
+import java.util.Scanner;
+
+/**
+ * The class to interact with users.
+ */
+public class Ui {
+    static final String LINE = "____________________________________________________________";
+
+    /**
+     * To welcome users.
```

Comment:
```suggestion
     * Welcomes users
```
Refer to https://se-education.org/guides/conventions/java/intermediate.html#comments on how to code Javadocs. Should start with "Welcomes", not "Welcome" :)